### ISSUE TITLE
Search in history

### ISSUE THREAD

**Issue by Leksat** [naotaka/ClipMenu#6]
Is there any plans for adding an ability to search through the history items?

**Comment by alexey-zaharchenko** [comment]
I have plans for add search since initial commit, but time goes by….

**Comment by chodorowicz** [comment]
Would be great! It's probably only feature I'm missing in ClipMenu.

**Comment by naotaka** [comment]
I'll start to make a prototype of ClipMenu in Swift language soon. I'll use Core Data as data models, so this feature would be included relatively easily.

**Comment by tenitski** [comment]
My ClipMenu is configured to remember 200 items and looking for something without search is a bit frustrating - another vote for this feature.

**Comment by naotaka** [comment]
I agree with you, @tenitski . As I also want this feature, next ClipMenu will have it. :)

**Comment by afreet** [comment]
I would also love to see this.  Are you looking for additional contributors?
